zuul.d/projects.yaml

```diff
@@ -31,3 +31,12 @@
               - ^lib/ansible/plugins/inventory/openstack.py
               - ^lib/ansible/module_utils/openstack.py
               - ^lib/ansible/utils/module_docs_fragments/openstack.py
+
+- project:
+    name: ansible/ansible
```

Comment:
We have added `ansible/ansible` project at line#7, only need to add `periodic` pipeline into it